src/typescript-service.ts

```diff
@@ -69,7 +69,7 @@ export class TypeScriptService implements LanguageHandler {
 
 	async initialize(params: InitializeParams, token = CancellationToken.None): Promise<InitializeResult> {
 		if (params.rootUri || params.rootPath) {
-			this.root = params.rootPath || util.uri2path(params.rootUri);
+			this.root = params.rootPath ? util.uri2path(params.rootPath) : util.uri2path(params.rootUri);
```

Comment:
But that is an error in the sourcegraph implementation, which is why I moved the workaround to the build server. I would like to keep it out of here because this is the public LS that is used by other clients that behave correctly